ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -728,6 +728,19 @@ tests:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-upi-xpn
   timeout: 6h0m0s
+- as: e2e-gcp-xpn-dedicated-dns-project
+  optional: true
+  run_if_changed: (gcp|google)
+  steps:
+    cluster_profile: gcp
+    env:
+      BASE_DOMAIN: regional.qe.gcp.devcluster.openshift.com
+      CREDENTIALS_MODE: Passthrough
+      CUSTOM_OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: registry.build06.ci.openshift.org/ci-ln-b9vkskb/release:latest
+      PRE_CREATE_PRIVATE_ZONE: "no"
+      PRIVATE_ZONE_PROJECT_TYPE: dedicated-dns-project
```

Comment:
suggest to move the ENV var setting into `openshift-e2e-gcp-xpn-dedicated-dns-project` install workflow.